### ISSUE TITLE
Visto de residência para acompanhamento familiar

### ISSUE THREAD

**Issue by mariojrrc** [nighto/brasil-portugal#92]
Hoje já é possível solicitar o visto de acompanhamento familiar (Ex: esposa e filhos) e sair do brasil já com visto/autorização de residência... evitando assim ter que ir como turista e solicitar o reagrupamento familiar. 

Acho que a informação na seção [Tipo de Visto](https://nighto.gitbook.io/brasil-portugal/tipos-de-visto) poderia ser atualizada. 

Ref visto VSF: https://www.vfsglobal.com/portugal/brazil/pdf/D-Visto-de-resid%C3%AAncia-para-acompanhamento.pdf

**Comment by andersonwarren** [comment]
Verdade, eu mesmo tive uma entrevista com um funcionário do consulado e ele me perguntou justamente isso.

**Comment by mariojrrc** [comment]
@PauloGoncalvesBH eu vou dar entrada na papelada no mês que vêm, dai crio o PR já com as informações da minha experiência.
